client/js/config/modals.js

```diff
@@ -22,6 +22,15 @@ export const createViewer = async viewer => {
   }
 };
 
+export const addCommentToEvent = async commentId => {
+  try {
+    //return await
```

Comment:
Please explain what is the purpose of this try catch block?

Reply:
deleted